iocage_lib/ioc_exec.py

```diff
@@ -38,18 +38,21 @@
 
 class IOCExec(object):
     """Run jexec with a user inside the specified jail."""
-    def __init__(self,
-                 command,
-                 uuid,
-                 path,
-                 host_user="root",
-                 jail_user=None,
-                 plugin=False,
-                 unjailed=False,
-                 skip=False,
-                 stdin_bytestring=None,
-                 su_env=None,
-                 callback=None):
+    def __init__(
+            self,
+            command,
+            uuid,
+            path,
```

Comment:
We can make this optional as well, and then do a sanity check if uuid is supplied that this also must be